spam-prevention.md

```diff
@@ -83,19 +92,23 @@ We want to defend against attackers that have the following capabilities:
 * they are running modified (malicious) versions of LN node implementations
 * they are able to quickly create many seemingly unrelated nodes
 * they may already have long-lived channels (good reputation)
+* they might probe in-real time channel balances to adjust their spams
 
 There are important properties of Lightning that we must absolutely preserve:
 
 * payer and payee's anonymity
 * trustless payments
-* decentralization
-* minimal (reasonable) barrier to entry
+* minimal (reasonable) barrier to entry as routing node
 * minimal overhead/cost for legitimate payments
+* minimal overhead to declare public paths to the network
+* discriminate unreasonably between different class of payments (e.g hodl HTLCs)
```

Comment:
Can you explain this one? I'm not sure if you mean we should penalize these types or payments or not.

Reply:
Reworded, lmk if good enough.

Reply:
Looks good!